website/src/components/EventCalendar/EventCalendar.test.jsx

```diff
@@ -24,23 +25,15 @@ const myEventsList = [
   },
 ];
 
-// mock moment to return the same seperate times so we always render the same
+// mock date to return the same time so we always render the same
 // calendar range
 
 
 describe('EventSummaryCollection', () => {
   it('Snapshot test of EventCalendar in the default agenda view', () => {
-    let mockNCalls = 0;
-    jest.mock('moment', () => () => ({
-      format: () => {
-        if (mockNCalls > 0) {
-          return '2018-12-03T23:55:00';
-        }
-        mockNCalls += 1;
-        return '2018-12-03T21:10:00';
-      },
-    }));
+    MockDate.set(1543801827000);
```

Comment:
wouldn't using Date.now() make more sense? because in use case date isn't constant. any reason for not using Date.now()? can tests not call on that?

Reply:
Yeah its very annoying we need to mock the current date, I tried to mock both Date.now and moment and none of them were working properly. Apparently moment uses different methods to get the current date across different versions. and react-big-calendar also does so the best approach i found was to use the mockdate library which handles mocking all the methods for dates.  Found it on stack overflow https://stackoverflow.com/questions/29719631/how-do-i-set-a-mock-date-in-jest